docs/features/init_scripts.md

```diff
@@ -0,0 +1,51 @@
+# :material-script: Init scripts
+
+[Init scripts](https://docs.databricks.com/clusters/init-scripts.html) is a powerful tool in Databricks to setup the workflow environment before the workflow is running. A very common use case is to [setup the Python pip.conf](https://learn.microsoft.com/en-us/azure/databricks/kb/clusters/install-private-pypi-repo) if the workflow needs some private packages, then you don't need to declare it in each [pip install](https://docs.databricks.com/libraries/notebooks-python-libraries.html#install-a-private-package-with-credentials-managed-by-databricks-secrets-with-pip).
```

Comment:
Please add SINCE 0.8.0 Badge before this section after header

Reply:
added now